apps/nextjs-website/src/editorialComponents/Feature/Feature.tsx

```diff
@@ -9,15 +9,18 @@ export interface FeatureProps {
   subtitle?: string;
   items: FeatureItem[];
   background?: string;
+  useDarkTheme?: boolean;
 }
 
 const Feature = (props: FeatureProps) => {
-  const { title, subtitle, items, background } = props;
+  const { title, subtitle, items, background, useDarkTheme = false } = props;
   const { palette } = useTheme();
   const theme = palette.mode;
 
-  const themeStyle = theme === 'light' ? 'text.primary' : 'background.paper';
-  const themeStyleBg = theme === 'light' ? 'background.paper' : 'primary.dark';
+  const themeStyle =
+    !useDarkTheme && theme === 'light' ? 'text.primary' : 'background.paper';
+  const themeStyleBg =
+    !useDarkTheme && theme === 'light' ? 'background.paper' : 'primary.dark';
```

Comment:
To keep the code DRY (Don't Repeat Yourself) and improve readability you could simplify this code introducing a third constant like that:

`const isDarkMode = useDarkTheme || theme !== 'light';

const textStyle = isDarkMode ? 'background.paper' : 'text.primary';
const imageStyle = isDarkMode ? 'background.paper' : 'primary.main';`